Tools/CMake/Config.cmake

```diff
@@ -0,0 +1,256 @@
+    set(IS_SIGNING 0)
+  endif()
+
+  if(${TIMESTAMP_AT_BUILD_TIME})
+    set(CODESIGN_TIMESTAMP_FLAG "--timestamp")
+  else()
+    set(CODESIGN_TIMESTAMP_FLAG "--timestamp=none")
+  endif()
+
+  option(INSTALL_R_FRAMEWORK "Whether to download and prepare R.framework" ON)
+
+  # Later we use this to produce two DMG file, or maybe a Universal one
+  # but I'm not yet so sure how that will work
+  if(CMAKE_OSX_ARCHITECTURES STREQUAL "x86_64")
+    set(DARWIN_ARCH "x86_64")
+    set(CPACK_ARCH_SUFFIX "Intel")
+  elseif(CMAKE_OSX_ARCHITECTURES STREQUAL "arm64")
+    set(CPACK_ARCH_SUFFIX "Apple")
+    set(DARWIN_ARCH "aarch64")
+  else()
+    set(CPACK_ARCH_SUFFIX ${CMAKE_OSX_ARCHITECTURES})
+  endif()
+
+  # Getting the major version of Xcode
+  # There is already a variable called XCODE_VERSION but that is only
+  # set when Xcode generator is used, so I had to get this myself
+  execute_process(
+    ERROR_QUIET
+    COMMAND xcodebuild -version
+    OUTPUT_VARIABLE XCODEBUILD_OUTPUT
+    OUTPUT_STRIP_TRAILING_WHITESPACE)
+
+  if(NOT (XCODEBUILD_OUTPUT STREQUAL ""))
+    string(
+      SUBSTRING ${XCODEBUILD_OUTPUT}
+                6
+                2
+                XCODE_VERSION)
+  else()
+    set(XCODE_VERSION "")
+  endif()
+
+  if(XCODE_VERSION STREQUAL "")
+    message(
+      WARNING
+        "Please set the Xcode version using XCODE_VERSION to be able to notarise your app, e.g., 13.2.1"
+    )
+  else()
+    message(STATUS "Xcode version: ${XCODE_VERSION}")
+  endif()
+
+  execute_process(
+    COMMAND uname -r
+    RESULT_VARIABLE result
+    OUTPUT_VARIABLE DARWIN_VERSION
+    OUTPUT_STRIP_TRAILING_WHITESPACE)
+
+  set(CONFIGURE_HOST_FLAG
+      ${CMAKE_OSX_ARCHITECTURES}-apple-darwin${DARWIN_VERSION})
+  message(STATUS "  ${CONFIGURE_HOST_FLAG}")
+
+endif()
+
+if(WIN32)
+
+  set(USE_CONAN ON)
+  set(SYSTEM_TYPE WIN32)
+
+  message(STATUS ${MSVC_TOOLSET_VERSION})
+  message(STATUS ${MSVC_VERSION})
+
+  set(VC_MERGE_MODULE_NAME
+      "Microsoft_VC143_CRT_x64.msm"
+      CACHE PATH "Module Merge Name")
+  if(MSVC_VERSION GREATER "1930")
+    set(VC_TOOLS_REDIST_DIR_VARIABLE "%VCINSTALLDIR%")
+    set(VC_VARS_PATH_NATIVE
+        "C:\\Program Files\\Microsoft Visual Studio\\2022\\Community\\VC\\Auxiliary\\Build"
+    )
+  elseif(MSVC_VERSION GREATER "1920")
+    set(VC_TOOLS_REDIST_DIR_VARIABLE "%VCToolsRedistDir%")
+    set(VC_VARS_PATH_NATIVE
+        "C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Community\\VC\\Auxiliary\\Build"
+    )
+  endif()
+
+endif()
+
+if(LINUX)
+
+  set(CUSTOM_R_PATH
+      ""
+      CACHE PATH "Path to your custom R installation")
+
+  option(LINUX_LOCAL_BUILD "Whether we are building inside the Build folder"
+         OFF)
+
+  option(FLATPAK_USED "Whether we are building for Flatpak" OFF)
+
+  # IS_LINUX_LOCAL_BUILD is a special variable that will be used in install-module.R
+  # and it is suppose to store R's TRUE or FALSE
+  set(IS_LINUX_LOCAL_BUILD TRUE)
+
+  if(LINUX_LOCAL_BUILD)
```

Comment:
So I think the other way round is more logical. FLATPAK_USED is already set anyway and it is the only scenario where you aren't building JASP locally atm.

Reply:
Done!